tfx_addons/feature_selection/example/modules/iris_module_file.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Supplement for iris species example with specifics feature modification.
+This module file will be used in the feature selection component example.
+"""
+
+from sklearn.feature_selection import SelectKBest as SelectorFunc
```

Comment:
Lint is complaining because these imports aren't used:

```
************* Module iris_module_file
tfx_addons/feature_selection/example/modules/iris_module_file.py:19:0: W0611: Unused SelectKBest imported from sklearn.feature_selection as SelectorFunc (unused-import)
tfx_addons/feature_selection/example/modules/iris_module_file.py:20:0: W0611: Unused f_classif imported from sklearn.feature_selection as ScoreFunc (unused-import)
```

Could you remove them?